### PR TITLE
fix #83610 www.nike.com

### LINKED ISSUE #83610: www.nike.com
### Issue URL (Annoyance)
 [https://www.nike.com/pl/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.nike.com%2Fpl%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880), <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media, <br/>Fanboy's Social Blocking List<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>I don't care about cookies<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> Official Polish filters for AdBlock, uBlock Origin & AdGuard, <br/>Polish Social Filters, <br/>KAD - Anti-Scam, <br/>Polish Annoyances Filters, <br/>Polish Anti Adblock Filters

### PR DESCRIPTION
**Pull request by stanislav-atr**
#83610

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -121,6 +121,7 @@ kraj.by#$#div[aria-labelledby="CModalLabel"] { display: none !important; }
 kraj.by#$#div[aria-labelledby="CModalLabel"] ~ div.modal-backdrop { display: none !important; }
 kraj.by#$#body { overflow: auto !important; padding-right: 0 !important; }
 sapatinhodeluxo.com.br##.bar-cookies
+nike.com#?##gen-nav-commerce-header-v2 > div.pre-modal-window:has(> .pre-modal > .pre-modal-view > .pre-cookie-modal-body)
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:124]: Broken page -> https://www.nike.com/pl/w/nowosci-mezczyzni-buty-3n82yznik1zy7ok

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt:124]: Try `nike.com#%#//scriptlet('set-cookie', 'sq', '3')`

**Review by zloyden — APPROVED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (krystian3w) [reply AnnoyancesFilter/sections/cookies_specific.txt]: broken was only scrolling?

Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Eh, this pull request was merged, but the scriptlet rule is ok here.
Have you found any issues with css rules? I mean
```
nike.com#?##gen-nav-commerce-header-v2 > div.pre-modal-window:has(> .pre-modal > .pre-modal-view > .pre-cookie-modal-body)
nike.com#$#body[data-overlay="cookieModalSimple"] { position: static !important; }
```

Comment (krystian3w) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Now I can't found issues with frozen scripts on pages (e.g. breakage to add product into "shop bag") or glitched pop-up based on node/DOM `.pre-modal-window` (aka glitch [AdguardTeam/ExtendedCss/issues/122](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fgithub.com%2FAdguardTeam%2FExtendedCss%2Fissues%2F122))

I suppose newsletter may block scroll too in categories but now I can't load to found used `[data-overlay]`.

Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Thank you, I'll change it to your rules in the future, if I find something wrong with the current rule.

Comment (krystian3w) [reply AnnoyancesFilter/sections/cookies_specific.txt]: ExntendedCSS may no longer safe:  #88423

**Pull request merged**